c8run/main.go

```diff
@@ -502,13 +502,7 @@ func main() {
 		if err != nil {
 			fmt.Print("Failed to write to file: " + connectorsPidPath + " continuing...")
 		}
-		var extraArgs string
```

Comment:
@jessesimpson36 we removed this as this =broke the camunda 8 setup, as it bypassed all the internal default configs see https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.files.

Reply:
I found older docs from spring where they kept the old config, it might have been caused by a recent update then.

Reply:
the new equivalent may be `spring.config.additional-location`
>If you prefer to add additional locations, rather than replacing them, you can use spring.config.additional-location. Properties loaded from additional locations can override those in the default locations.

Reply:
Would you prefer to migrate to the new property, as removing it would be a regression? Happy to test that then.

Reply:
I would like that, but it doesn't need to be part of this PR if you'd rather merge this first. the `--config` option is pretty hard to test (at least on windows)

Reply:
Yeah, happy to assist on this in a follow-up if you would add it back, would keep it out of scope here under the assumption this is an acceptable regression from your perspective.

Reply:
This is an acceptable regression. :)